networking_cisco/tests/unit/cisco/cfg_agent/test_aci_asr1k_cfg_syncer.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2014 Cisco Systems, Inc.  All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from oslo_config import cfg
+from oslo_serialization import jsonutils
+
+import mock
+from networking_cisco.tests import base
+from networking_cisco.tests.unit.cisco.cfg_agent import (
+    test_asr1k_cfg_syncer as test_sync)
+
+
+from networking_cisco.plugins.cisco.cfg_agent.device_drivers.asr1k import (
+    aci_asr1k_cfg_syncer)
+
+
+class AciASR1kCfgSyncer(test_sync.ASR1kCfgSyncer):
+
+    def _read_neutron_db_data(self):
+        """
+        helper function for reading the dummy neutron router db
+        """
+        with open(base.ROOTDIR +
+                  '/unit/cisco/etc/cfg_syncer/neutron_router_db_aci.json',
```

Comment:
do we really need to add this file into this patch then load it up here? In the UTs below you always make the DB empty so it doesn't seem like we need this new file at all....